CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `Ctl.Examples.PlutusV2.OneShotMinting` example, demonstrating the use of `applyArgs` with a PlutusV2 script ([#1038](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1038))
 - New `Contract.Test.Utils` assertions and checks: `assertOutputHasRefScript`, `checkOutputHasRefScript`, `checkTxHasMetadata` ([#1044](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1044))
 - `Parallel` instance to `Contract` monad. Parallel capabilities are in the associated `ParContract` datatype ([#1037](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1037))
+- `Contract.Cbor.hexToCborBytes` and `Contract.Cbor.hexToCbor` to decode a hexadecimal string to a `CborBytes` or `Cbor`, respectively. ([#850](https://github.com/Plutonomicon/cardano-transaction-lib/issues/850))
```

Comment:
I'm not sure if there's a reason I'm not aware of, but is it worth it to export both of these? I think we could just export a function to work with `Cbor` and keep `CborBytes` for internal use. I think users will wonder what the difference is between the two types, then they'll need to read through the documentation/source more closely to figure it out. They can just use `unwrap` to get the `CborBytes` if that's really necessary

Reply:
I agree. It has been removed.